pom.xml

```diff
@@ -205,57 +264,34 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-compiler-plugin</artifactId>
-        <version>2.5.1</version>
         <configuration>
-          <encoding>UTF-8</encoding>
           <source>${java.target}</source>
           <target>${java.target}</target>
         </configuration>
       </plugin>
-      <!-- we need to create a uber-jar for https://github.com/ktoso/maven-git-commit-id-plugin/issues/196 -->
+
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-shade-plugin</artifactId>
-        <version>2.3</version>
+        <artifactId>maven-plugin-plugin</artifactId>
         <executions>
           <execution>
-            <phase>package</phase>
+            <id>default-descriptor</id>
+            <phase>process-classes</phase>
```

Comment:
(task for me during release notes basically, for my reference https://maven.apache.org/guides/introduction/introduction-to-the-lifecycle.html )